imports/plugins/core/checkout/client/components/completedOrder.js

```diff
@@ -18,95 +20,228 @@ import AddEmail from "./addEmail";
-        {/* show a different message depending on whether we have an email or not */}
-        <AddEmail order={order} orderEmail={order.email} />
-        {/* This is the left side / main content*/}
-      </div>
-    );
-  }
+    if (!this.state.order) {
+      return (
+        <Components.NotFound
+          i18nKeyTitle="order.notFound"
+          icon="fa fa-barcode"
+          title="Order Not Found"
+        />
+      );
+    }
 
-  return (
-    <div className="container order-completed">
-      { headerText }
-      <div className="order-details-main">
-        <div className="order-details-content-title">
-          <p><Components.Translation defaultValue="Your Items" i18nKey={"cartCompleted.yourItems"} /></p>
-        </div>
-        {shops.map(function (shop) {
-          const shopKey = Object.keys(shop);
-          return (
-            <CompletedShopOrders
-              shopName={shop[shopKey].name}
-              items={shop[shopKey].items}
-              key={shopKey}
-              shippingMethod={shop[shopKey].shippingMethod}
-              handleDisplayMedia={handleDisplayMedia}
-              isProfilePage={isProfilePage}
+    if (this.state.isProfilePage) {
+      headerText = (
+        <p className="order-id">
+          <strong>Order ID</strong>
+          {this.state.orderId}
+        </p>
+      );
+    } else {
+      headerText = (
+        <div className="order-details-header" > { /* This is the left side / main content */}
+          <h3 >
+            <Components.Translation
+              defaultValue="Thank You"
+              i18nKey="cartCompleted.thankYou"
             />
-          );
-        })}
-      </div>
-
-      <div className="order-details-side">
+          </h3>
+          <p>
+            <strong>Order ID </strong>
+            {this.state.orderId}
+          </p> { /* show a different message depending on whether we have an email or not */}
+          <AddEmail
+            order={this.state.order}
+            orderEmail={this.state.order.email}
+          /> { /* This is the left side / main content */}
+          {this.state.isCancelled ? (<p className="completed-cancel">Your order has been cancelled</p>) : " "}
+        </div>
+      );
+    }
+    return (
+      <div className="container order-completed" >
+        {headerText}
+        <div className="order-details-main" >
+          <div className="order-details-content-title" >
+            <p>
+              <Components.Translation
+                defaultValue="Your Items"
+                i18nKey="cartCompleted.yourItems"
+              />
 
-        {/* This is the right side / side content */}
-        <div className="shipping-payment-details">
-          <div className="shipping-info">
-            <div className="order-details-content-title">
-              <p> <Components.Translation defaultValue="Shipping Address" i18nKey={"cartCompleted.shippingAddress"} /></p>
+            </p >
+          </div> {
```

Comment:
Move the opening brace to a new line instead of having it after the closing `div` tag